it/auth/IndividualLossesApiAuthMainAgentsOnlyISpec.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package auth
+
+import api.services.DownstreamStub
+import play.api.http.Status
+import play.api.libs.json.{JsValue, Json, OWrites, Writes}
+import play.api.libs.ws.{WSRequest, WSResponse}
+import v4.models.domain.lossClaim.TypeOfClaim
+import v5.lossClaims.amendOrder.def1.model.request.Claim
+
+class IndividualLossesApiAuthMainAgentsOnlyISpec extends AuthMainAgentsOnlyISpec {
+
+  val callingApiVersion = "5.0"
+
+  val supportingAgentsNotAllowedEndpoint = "amend-loss-claims-order"
+
+  val mtdUrl = s"/$nino/loss-claims/order/2023-24"
+
+  def sendMtdRequest(request: WSRequest): WSResponse = await(request.put(requestJson()))
+
+  val downstreamUri = s"/income-tax/claims-for-relief/preferences/23-24/$nino"
+
+  override val downstreamHttpMethod: DownstreamStub.HTTPMethod = DownstreamStub.PUT
+
+  val maybeDownstreamResponseJson: Option[JsValue] = None
+
+  override protected val downstreamSuccessStatus: Int = Status.NO_CONTENT
+
+  def requestJson(): JsValue = {
+    val claim1: Claim                 = Claim("1234567890ABEF1", 1)
```

Comment:
These don't really need to specify the type (and Intellij doesn't object if the type isn't there), so could remove the ": Claim" etc. Except for the two Writes, as it's useful there.